dvc/stage.py

```diff
@@ -251,15 +260,19 @@ def _changed_outs(self):
 
         return False
 
-    def _changed_md5(self):
-        if self.changed_md5():
+    def _changed_checksum(self):
+        if self.changed_checksum():
             logger.warning("Dvc file '{}' changed.".format(self.relpath))
             return True
         return False
 
     def changed(self):
         ret = any(
-            [self._changed_deps(), self._changed_outs(), self._changed_md5()]
+            [
+                self._changed_deps(),
+                self._changed_outs(),
+                self._changed_checksum(),
+            ]
         )
```

Comment:
Using ors instead of `any()` here will simplify code and provide short-circuit behavior.